src/_pytest/pytester.py

```diff
@@ -361,9 +362,9 @@ class RunResult:
     :ivar duration: duration in seconds
     """
 
-    def __init__(self, ret, outlines, errlines, duration):
+    def __init__(self, ret: Union[int, ExitCode], outlines, errlines, duration) -> None:
         try:
-            self.ret = pytest.ExitCode(ret)
+            self.ret = pytest.ExitCode(ret)  # type: Union[int, ExitCode]
```

Comment:
Required to fix `mypy: Incompatible types in assignment (expression has type "Union[int, ExitCode]", variable has type "ExitCode")` below on line 369 - makes sense?

Reply:
Yes, makes perfect sense.